ceph.spec.in

```diff
@@ -249,8 +248,9 @@ BuildRequires:  openldap-devel
 BuildRequires:  openssl-devel
 BuildRequires:  CUnit-devel
 BuildRequires:  redhat-lsb-core
+%if 0%{?rhel} == 7
+BuildRequires:	btrfs-progs
```

Comment:
IIRC, @tchaikov has a commit that removes/changes this dependency/moves it to 'make check' deps.

Reply:
lgtm, Kefu can comment more on the btrfs-progs package.

Reply:
@ktdreyer @b-ranto yes, i moved that commit into a separated PR at #26526 .